go.mod

```diff
@@ -1,84 +1,86 @@
 module github.com/kubeflow/training-operator
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.2
 
 require (
 	github.com/go-logr/logr v1.4.1
 	github.com/google/go-cmp v0.6.0
-	github.com/onsi/ginkgo/v2 v2.14.0
-	github.com/onsi/gomega v1.30.0
+	github.com/onsi/ginkgo/v2 v2.17.2
+	github.com/onsi/gomega v1.33.1
 	github.com/open-policy-agent/cert-controller v0.10.1
-	github.com/prometheus/client_golang v1.18.0
+	github.com/prometheus/client_golang v1.19.0
 	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.4
-	go.uber.org/zap v1.26.0
-	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v0.29.3
-	k8s.io/code-generator v0.29.3
-	k8s.io/klog/v2 v2.110.1
-	k8s.io/kube-openapi v0.0.0-20231010175941-2dd684a91f00
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
-	sigs.k8s.io/controller-runtime v0.17.2
+	github.com/stretchr/testify v1.9.0
+	go.uber.org/zap v1.27.0
+	k8s.io/api v0.30.0
+	k8s.io/apimachinery v0.30.0
+	k8s.io/client-go v0.30.0
```

Comment:
@franciscojavierarceo Bumping k8s version will potentially bring us breaking changes.
So, could you avoid bumping the k8s versions during code freeze?

Reply:
I guess that we need to bump these libs to CVE fixed version. Your command indicates that libs are upgraded to latest version. I meant that we should not bump libs to the latest in specifically, K8s.

> go get -u k8s.io/apiextensions-apiserver
go get -u sigs.k8s.io/controller-runtime
go get -u volcano.sh/apis